### ISSUE TITLE
[Blazor] Constructor injection example - drop property usage

### ISSUE THREAD

**Issue by hakenr** [dotnet/AspNetCore.Docs#33530]
### Description

```csharp
public partial class ConstructorInjection(NavigationManager navigation)
{
    protected NavigationManager Navigation { get; } = navigation;
}
```

Do we still want to use a property when demonstrating constructor injection?

I think we've gotten used to seeing properties with injected services in all the Blazor samples so far. However, to be honest, that's just a result of the fact that Blazor didn’t support constructor injection until recently. When using standard constructor injection, we typically don't create properties for services. Instead, we use them as `private readonly` fields or, somewhat controversially, capture them directly from the primary constructor.

I think an example like this would be more appropriate:
```csharp
public partial class MyComponent(NavigationManager navigation)
{
    private void HandleClick()
    {
        navigation.NavigateTo(newUri);
    }
}
```

Can I update the example to reflect this approach?

cc @guardrex 

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/blazor/fundamentals/dependency-injection?view=aspnetcore-9.0#request-a-service-in-a-component

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/blazor/fundamentals/dependency-injection.md

### Document ID

d9aabe95-69e9-0a03-81ff-429914137f9d

### Article author

@guardrex

**Comment by hakenr** [comment]
(The history repeats itself - it was the same with WebForms and MVC, where they initially only had `[Inject]` as well.)

**Comment by guardrex** [comment]
Yes, it's a bit stale. I'll get back to you on this tomorrow. I'm off to 🛌💤 soon.

**Comment by guardrex** [comment]
I'm UP 🥐☕.

Yes, please proceed.

BTW ... Would you please fix a misspelled word 😈 on Line 619? ...

```diff
- BlazorServiceAcccessor
+ BlazorServiceAccessor
```

The versioning for the updated code is .NET 8 or later ...

```markdown
:::moniker range=">= aspnetcore-8.0"



:::moniker-end

:::moniker range="< aspnetcore-8.0"



:::moniker-end
```

Please add a cross-link to the *Additional resources* section (.NET 8 or later block) for ...

```markdown
* [Tutorial: Explore primary constructors](/dotnet/csharp/whats-new/tutorials/primary-constructors)
```
